lib/percona_migrator/runner.rb

```diff
@@ -0,0 +1,51 @@
+require 'open3'
+
+module PerconaMigrator
+  class Runner
+
+    NONE = "\e[0m"
+    CYAN = "\e[38;5;86m"
+    GREEN = "\e[32m"
+    RED = "\e[31m"
```

Comment:
What about using this? https://github.com/fazibear/colorize it would even allow you to activate/deactivate colorization.